src/docs/doxygen/Doxyfile.in

```diff
@@ -58,7 +58,7 @@ PROJECT_LOGO           =
 # entered, it will be relative to the location where doxygen was started. If
 # left blank the current directory will be used.
 
-OUTPUT_DIRECTORY       = ./html
+OUTPUT_DIRECTORY       = @PROJECT_BINARY_DIR@/src/docs/html/doxygen
```

Comment:
Did this work? I vaguely remember Axom having to rip out any cmake variables because readthedocs wouldnt run cmake over the file.

Reply:
Yep. I am following the axom example of just adding a redefinition of this variable at the end of the file in the `conf.py` script.